### PR TITLE
improve dockerignore / gitignore / schema dumping

### PR DESCRIPTION
**Pull request by grosser**
previously images were pushed to github with whatever local config/database.yml the builder had
which would most likely not work for anyone else or worst-case could even include remote db credentials

@zendesk/bre @dndungu 
/cc @zendesk/compute

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
-node_modules
+
+# user might chose to have local gem installs, they can be quiet big
 vendor/bundle
 
-public/assets
+# editor-specific folders that can be big
+.idea
+
+## copied from .gitignore
```
Comment (adammw) [inline .dockerignore]: doesn't dockerconfig support including the gitignore? why copy it?

Comment (grosser) [reply .dockerignore]: nope

Comment (adammw) [reply .dockerignore]: I must be thinking of the gcloud ignore...

**Comment by grosser** [comment]
yep

**Pull request merged**